pages/index.js

```diff
@@ -1,5 +1,7 @@
-import CardList from '../src/components/CardMap/CardList';
 import getCardMap from '../src/components/services/get-cardmap';
+import Form from '../src/components/Form/Form';
+import dynamic from 'next/dynamic';
+const CardList = dynamic(() => import('../src/components/CardMap/CardList'), { ssr: false });
 
 export function getStaticProps() {
 	const data = getCardMap();
```

Comment:
I think you don't need those static props anymore since you get the card data from zustand.